pkg/plugin/linuxutil/ethtool_stats_linux.go

```diff
@@ -28,11 +29,13 @@ func NewEthtoolReader(opts *EthtoolOpts, ethHandle EthtoolInterface) *EthtoolRea
 			return nil
 		}
 	}
+	// Construct a cached ethtool handle
+	CachedEthHandle := NewCachedEthtool(ethHandle, opts)
 	return &EthtoolReader{
 		l:         log.Logger().Named(string("EthtoolReader")),
 		opts:      opts,
 		data:      &EthtoolStats{},
```

Comment:
can we remove data struct here since all ethtool stats are coming from handle? not blocking, but maybe future consideration